test_validators.py

```diff
@@ -18,6 +19,25 @@ def test_invalid_uuid(uuid):
     assert not verify_uuid_format(uuid)
 
 
+@pytest.mark.parametrize("satellite_id", ["4a8fb994-57fe-4dbb-ad2a-9e922560b6c1", "1000056432"])
```

Comment:
```suggestion
@pytest.mark.parametrize(
    "satellite_id", ["4a8fb994-57fe-4dbb-ad2a-9e922560b6c1", "4a8fb99457fe4dbbad2a9e922560b6c1", "1000056432"]
)
```

A UUID without hyphens is ok too. Adding one doesn’t hurt.